index.html

```diff
@@ -210,10 +211,11 @@ <h2 class="footer-intro">I'm always interested in hearing about new projects, so
             placeholder="Enter text"
             required></textarea></li>
         </ul>
-        <button type="submit"> Get it touch</button>
+        <button class="submit-form"type="submit"> Get it touch</button>
+      </div>
       </form>
-      <div>
-        <ul class="socialmediaicons">
+      <div class="form-bar">
+        <ul class="socialmediaicons-form">
           <li><a href="https://github.com/Cluab"><i class="fa-brands fa-github"></i></a></li>
           <li><a href="https://linkedin.com/in/Ebrahim Al-Yousefi"><i class="fa-brands fa-linkedin"></i></a></li>
           <li><a href="https://www.instagram.com/27.i.b/"><i class="fa-brands fa-instagram"></i></a></li>
```

Comment:
- [ OPTIONAL] I would again recommend you to change your icons to be the same as those on Figma. Because this is among the requirement. The previous reviewer just forget to point that out. 
Your design 👇️
![Screenshot from 2022-09-27 16-26-53](https://user-images.githubusercontent.com/89228743/192582301-6e755131-f328-4789-a471-c6af7205ed41.png)

The Figma design 👇️
![Screenshot from 2022-09-27 16-28-53](https://user-images.githubusercontent.com/89228743/192582711-f7061cf0-87ac-4d18-aee5-2890b42c5da9.png)